mobile-app/app/tailwind.config.js

```diff
@@ -179,7 +179,9 @@ module.exports = {
         0.5: 0.5
       },
       borderRadius: {
-        '2lg': 10
+        'lg-v2': 10,
+        '2lg-v2': 15,
+        'xl-v2': 20
```

Comment:
can we use this instead:
`xl-v2`: 15, and `2xl-v2`: 20

Reply:
Sure, I will make changes